Patch-Source: https://github.com/xbmc/xbmc/pull/27064
---
From 216d14755dd9ae46f3f098ebcb0589209574d651 Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Fri, 8 Aug 2025 15:02:45 +0200
Subject: [PATCH] [xbmc] Add missing include <stdint.h>

Starting with gcc 15 it failed on musl libc with:

	/home/buildozer/git/aports/community/kodi/src/xbmc-21.2-Omega/xbmc/guilib/GUIMessage.h:378:15: error: unknown type name 'int64_t'
	  378 |               int64_t param1,
	      |               ^
	...

Signed-off-by: Achill Gilgenast <achill@achill.org>
---
 xbmc/guilib/GUIListItem.h | 1 +
 xbmc/guilib/GUIMessage.h  | 2 ++
 xbmc/video/VideoUtils.h   | 1 +
 3 files changed, 4 insertions(+)

diff --git a/xbmc/guilib/GUIListItem.h b/xbmc/guilib/GUIListItem.h
index 429a31af3b..53668c58c7 100644
--- a/xbmc/guilib/GUIListItem.h
+++ b/xbmc/guilib/GUIListItem.h
@@ -15,6 +15,7 @@
 
 #include <map>
 #include <memory>
+#include <stdint.h>
 #include <string>
 
 //  Forward
diff --git a/xbmc/guilib/GUIMessage.h b/xbmc/guilib/GUIMessage.h
index 18e605c811..2f44374732 100644
--- a/xbmc/guilib/GUIMessage.h
+++ b/xbmc/guilib/GUIMessage.h
@@ -8,6 +8,8 @@
 
 #pragma once
 
+#include <stdint.h>
+
 /*!
 \file GUIMessage.h
 \brief
diff --git a/xbmc/video/VideoUtils.h b/xbmc/video/VideoUtils.h
index e6120284f1..f609430866 100644
--- a/xbmc/video/VideoUtils.h
+++ b/xbmc/video/VideoUtils.h
@@ -11,6 +11,7 @@
 #include "utils/ContentUtils.h"
 
 #include <memory>
+#include <stdint.h>
 
 class CFileItem;
 class CFileItemList;
-- 
2.50.1

